app/controllers/wiki_controller.rb

```diff
@@ -148,7 +148,14 @@ def create
         end
         redirect_to @node.path
       else
-        render action: :edit
+        flash.now[:notice] = "This is the new rich editor. For the legacy editor, <a href='/post?#{request.env['QUERY_STRING']}&legacy=true' class='legacy-button'>click here</a>."
```

Comment:
Great -- here, we'll want to link to `/wiki/new?legacy=true` and then have a conditional letting people use the original template, on the line above. Make sense?